develdocs.phtml

```diff
@@ -68,11 +68,11 @@ include("externals/menu.phtml");
 </div>
 <p class="justify"><?php echo T_("Get a copy of the documentation source as described above and start making changes. When you are satisfied with your changes you can create a patch by running the following command in the base directory of the documentation (usually gnucash-docs, unless you renamed it):");?></p>
 <pre class="command">git diff > my-patch-name.patch</pre>
-<p class="justify"><?php echo T_("Next attach your patch to a bug report against the proper component for the Documentation product in <a href='https://bugs.gnucash.org/'><span class=\"gnucash\">GnuCash</span>' bug database</a>. See also <a href='https://wiki.gnucash.org/wiki/Bugzilla'>GnuCash' wiki page on bugzilla</a> for more details about patch submission.");?></p>
+<p class="justify"><?php echo T_("Next, attach your patch to a bug-report against the proper component for the <b>Documentation</b> product in the <a href='https://bugs.gnucash.org/'><span class=\"gnucash\">GnuCash</span>' bug database</a>. See also <a href='https://wiki.gnucash.org/wiki/Bugzilla'>GnuCash' wiki page on Bugzilla</a> for more details about patch submission.");?></p>
 <div class="dialog note">
-<p class="justify"><?php echo T_("<b>NOTE:</b> It used to be ok also to send your patch directly to the gnucash-devel list. This is discouraged now, as a patch is easily forgotten between the many list discussions. Attach patches to bugs in Bugzilla instead (either an existing bug or a new one). If you insist on sending a patch to gnucash-devel, it should be attached, not inlined.");?></p>
+<p class="justify"><?php echo T_("<b>Note:</b> It used to be OK to also send your patch directly to the gnucash-devel list. This is discouraged now, as a patch is easily forgotten between the many list discussions. Attach patches to bugs in Bugzilla instead (either an existing bug or a new one). If you insist on sending a patch to gnucash-devel, it should be attached, not inlined.");?></p>
```

Comment:
```suggestion
<p class="justify"><?php echo T_("<b>Note:</b> Previously, patches could be sent directly to the gnucash-devel list. This is now discouraged, as those are easily forgotten between the many list discussions. Attach patches to bug reports in Bugzilla instead (either in an existing or new bug report). If you insist on sending your patch to gnucash-devel, attach it instead of inlining it.");?></p>
```